recipes/uppasd/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "uppasd" %}
+{% set version = "v6.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UppASD/UppASD/releases/download/{{ version }}/{{ version }}.tar.gz
+  sha256: f483bb6e0a5c75820e4a3d38894c40557548777160eabd3c152c8c356e6ca8ed
+  patches:
+    - binary-name.patch
+    - cmake.patch
+    - cmake-windows.patch  # [win]
+    - openmp.patch  # [win]
+    - transfer-size.patch  # [win]
+    - format-error.patch  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('fortran') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - ninja
+  host:
+    - fftw  # [not win]
```

Comment:
IIUC from all the notifications, something with fftw wasn't working on windows?

Reply:
I managed to switch to MKL directly by restricting it to sufficiently new MKL versions, that depend on llvm-openmp instead of intel-openmp. @h-vetinari Would this be acceptable or does it impose too strict restrictions, resulting in the package not being usable in combination with many others?

Would further changes/pinning be required if https://github.com/conda-forge/intel_repack-feedstock/pull/119 gets merged?

Reply:
> Would this be acceptable or does it impose too strict restrictions, resulting in the package not being usable in combination with many others?

Yes, MKL 2025 is acceptable.

> Would further changes/pinning be required if conda-forge/intel_repack-feedstock#119 gets merged?

Shouldn't change anything, especially since you're depending on `llvm-openmp` here.